### ISSUE TITLE
Definitions of alpha and gamma in Moffat functions

### ISSUE THREAD

**Issue by rjavila** [astropy/astropy#7733]
The documentation in the Moffat1D and Moffat2D models use alpha and gamma, but as far as I can tell, everywhere else these two parameters are defined as beta and alpha, respectively. For example equation 1 in this paper https://arxiv.org/pdf/astro-ph/0109067.pdf

Can the code and documentation be changed to reflect this? It has confused me bit while using the models. Thanks.

**Comment by larrybradley** [comment]
That's unfortunate.  We can't simply swap them because it would break existing code that uses them.  We could possibly deprecate the current `alpha` and `gamma` parameters, renaming them to some intermediate names, and then deprecate the intermediate names in favor of the correct `alpha` and `gamma` parameters.  That process would require at least two release cycles.

**Comment by bsipocz** [comment]
IMO relying on names of variables can cause problems on many levels, but if the documentation clearly states what each parameter means then this ship has sailed.

**Comment by nden** [comment]
I think it's going to be very confusing to change the names now since the model has been around for about 5 years. 
:-1:

**Comment by rjavila** [comment]
Yeah, I understand that it would be a lot of work to make the code changes. I would say though that the definitions of `alpha` and `beta` have been around much longer than 5 years and this can cause confusion for a user that is expecting parameters to be defined in some way. No one would be ok if we started using Phi instead of sigma for standard deviation right? 

How about a warning or notification in the documentation to let users know if the difference?

**Comment by nden** [comment]
I am concerned about the impact such a change would have on other code written in these 5 years. It's also unclear to me why those names were chosen to start with. These two models started as `Beta` models and were renamed to `Moffat` later but the original parameter names were kept. Is there a context in which the current names are correct or was this an oversight?

**Comment by rjavila** [comment]
@eteq solution of a hack would be sufficient for me as a user. I'll let you decide what to do about that.

On the documentation side, I'd be happy to write something up.

**Comment by pllim** [comment]
Re: hack -- I attemped to "hack" Gaussian model once and got rejected (#4936). Should be we consistent in the no-hack policy? 😉

**Comment by hamogu** [comment]
We might make a new model `Moffat1DWithBetterParameterNames`. That model only needs a few lines of code - just call `Moffat1D` with the right parameters. The very existence of the second model will make users check the docs when they see it and then the dos will have to explain the difference. That way, current users of `Moffat1D` don't have to do anything and new users who want to use the right names simply use the new model. It's more duplication than the hack suggested above, but has much less potential for confusing the parameters.

**Comment by larrybradley** [comment]
@hamogu  I would even go further and suggest that after adding the "correct/better" model, that we deprecate  `Moffat1D/2D`.   (insert *runs for cover* emoji here) :smile:
